raspi/README.md

```diff
@@ -0,0 +1,232 @@
+file with your regular programming editors, like Atom):
+
+    ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+    update_config=1
+    country=US
+
+    network={
+        ssid="«your_SSID»"
+        psk="«your_PSK»"
+        key_mgmt=WPA-PSK
+    }
+
+Remember to change the `ssid` line with the name of the Wireless network to
+connect, and `psk` is its password. While you are at it, create an empty file in
+this directory called `ssh`.
+
+When you are done, un-mount the card with the following Terminal command:
+
+    $ sudo umount /media
+
+Take your MicroSD card from the reader, and back into a Raspberry Pi. When you
+power it on, it will connect to your network, and allow you to log into it over
+the network. The problem is, you need to know its *numeric networking address*.
+
+The Adafruit company has made a nice program called [Pi Finder][pif]. This
+program scans the network looking for Raspberry Pi, and allows you to open a
+**Terminal** on the Pi (assuming you created the `ssh` file mentioned above) and
+**Upload** files from your computer to it. Click the **Bootstrap** button to
+install some programs (see [these details][pf2]).
+
+  [pif]: https://github.com/adafruit/Adafruit-Pi-Finder
+  [pf2]: https://learn.adafruit.com/the-adafruit-raspberry-pi-finder/overview
+
+Install our software project on the system
+------------------------------------------
+
+Using either the **Terminal** button on the *Pi Finder* application, or starting
+up the **Terminal** on your laptop and typing:
+
+    $ ssh pi@10.0.1.187  # Or whatever the IP address is
+
+Once you are logged into the system, let's first secure the system by changing the password:
+
+    $ passwd
+    Changing password for pi.
+    (current) UNIX password:       # This is probably raspberry
+    Enter new UNIX password:       # Set to pigmice2733
+    Retype new UNIX password:
+    passwd: password updated successfully
+
+Next, let's install `git` and the other dependencies:
+
+    $ sudo apt-get install -y git python-pip python-opencv
+    $ sudo apt-get install -y python-numpy python-yaml
+    $ sudo pip install pynetworktables
+
+Now, we can get the latest version of our code on to the system:
+
+    $ git clone https://github.com/pigmice2733/pigmice-vision
+
+Once you `cd` into the project, test it out:
+
+    $ cd pigmice-vision
+    $ pip install -r requirements.txt
+
+Remember, you will need to have a valid configuration file, so you may want to
+start with the one on your laptop. To copy this, go to the **Terminal** on your
+laptop, and enter:
+
+    $ scp .pigmice-config.yaml pi@10.0.1.187:
+
+Static IP Address
+------------------
+We will be setting the static (not changing) ip address through the pi's dhcp
+file. To change this, nano (this allows us to edit a file through the command
+line) into the file.
+After ssh-ing into the pi, type this command in the command prompt:
+    $ sudo nano /etc/dhcpcd.conf
```

Comment:
I believe that markdown won't render this code without a blank line in front of it.